crates/proto/src/xfer/mod.rs

```diff
@@ -33,25 +33,29 @@ use crate::runtime::Time;
 
 #[cfg(feature = "std")]
 mod dns_exchange;
+#[cfg(feature = "std")]
```

Comment:
Nit: while you're in here (and potentially in a separate commit), would be nice to redo this to group imports with their module declaration and get rid of the `self::` prefixes everywhere.